### ISSUE TITLE
page builder is deactivating on it's own

### ISSUE THREAD

**Issue by direggio** [ThemeFuse/Unyson#1216]
Hi, I created a wordpress site for a few of my clients using the unyson plugin. For some strange reason every other day the page builder on the unyson plugin deactivates by itslef leaving a bunch of code on the website pages which is not good. This problem has been going on for some time now, please help me fix this problem from happening again. Thank you

**Comment by moldcraft** [comment]
Hi,

Which Unyson version do you use?

**Comment by direggio** [comment]
it says unyson version 2.4.17

**Comment by direggio** [comment]
![screen shot 2016-01-20 at 3 29 06 pm](https://cloud.githubusercontent.com/assets/16804735/12462142/affded60-bf8a-11e5-9cad-2f0a7643b664.png)

**Comment by moldcraft** [comment]
Try [this](https://gist.github.com/moldcraft/1fe4c3cded8536c66a19) to reproduce the problem

**Comment by direggio** [comment]
I tried but I can't seem to fix it if I give my wordpress login would you be able to help?

**Comment by moldcraft** [comment]
Send the details to `moldcraft(at)email.com`

**Comment by moldcraft** [comment]
I can't reproduce the problem. I tried to start plugin auto update, but it won't start. It seems that it's not enabled, `echo apply_filters( 'auto_update_plugin', 'hi');` returns `'hi'`, but if auto update is enabled, it should return `true`. Also I decreased plugin version and updated it from admin page, after update the extensions are there and are active.

> page builder is deactivating

So it is deactivating not uninstalling?
